lib/voteman.py

```diff
@@ -0,0 +1,195 @@
+"""
+実際に投票やメッセージの管理を行う
+メッセージ一つにつきインスタンス一つ
+"""
+import datetime
+import re
+
+import discord
+
+VOTE_START_MESSAGE = """\
+***† Twitter Judgement System †***
+<@&711190816122470450> 各位
+***{}***さんがツイートの可否の投票を開始しました！
+OK! → <:AC:693007620159832124>
+NG! → <:WA:693007620201775174>
+
+***<† 本文 †>***
+```
+{} 
+```
+***<† 情報 †>***
+```
+Vote Started By:†{} ({})†
+Content By:†{} ({})†  
+```
+"""
+
+ORIGINAL_EMOJI_REGEX = re.compile(r'<.*:\d*>')
+
+
+class MessageManager:
+    def __init__(self, vote_starter_message: discord.message, tweet_body_message: discord.message = None):
```

Comment:
```suggestion
    def __init__(self, vote_starter_message: discord.Message, tweet_body_message: discord.Message = None):
```
ぼくはめんどくさい人なので気になってしまいました…